### DISCUSSION TITLE
v3.2.2 - Export setup & error

### DISCUSSION THREAD

**Discussion by scottjo** [filamentphp/filament#10836] [Help] [answered]
Morning all

I have updated to v3.2.2 and I am trying to setup the export feature on one of my resources.

I believe that I have followed the instructions -  
* I've migrated the required tables
* I've created an exporter using the artisan command with --generate
* I've added it to the headerActions of the table definition

```
return $table
            ->striped()
            ->columns(
                self::getPropertyTableLayout()
            )
            ->filters([
                //
            ])
            ->actions([
                EditAction::make()->label(''),
            ])
            ->headerActions([
                ExportAction::make()
                    ->exporter(PropertyExporter::class)
            ])
            ->bulkActions([
                BulkActionGroup::make([
                    DeleteBulkAction::make(),
                ]),
            ]);
```

Now when I go and visit the page - I get an error

Method Filament\Actions\ExportAction::table does not exist.

What do I need to be looking out for?

**Comment by leandrocfe** [comment, marked as answer]
maybe `\Filament\Tables\Actions\ExportAction::make()`

**Comment by scottjo** [comment]
Thanks that sorted it
